### DISCUSSION TITLE
Add Affiliate Links?

### DISCUSSION THREAD

**Discussion by goaround** [towfiqi/serpbear#51] [Ideas]
Both ScrapingAnt and scrapingrobot offers an affiliate program with a lifetime commission. I think this would be a great way to support this project. Add your links to the docs and in the Settings screen @towfiqi.

https://scrapingant.com/legal/affiliate/
https://scrapingrobot.com/affiliate/

**Comment by nu-ad** [comment]
I for one would gladly purchase scrapers via affiliate link of SERPBEAR.

Very good work deserves recognition in any and all form. If there is any other form of support for this project, please do let us know.
